### PR TITLE
fixed with simple array input functions

### PR DESCRIPTION
**Pull request by soumyaniljana**
fixed few of the functions with right conventions and right return from function

### CODE REVIEW

File: Arrays/arraySample.sol [hunk 1]
```diff
@@ -3,8 +3,8 @@ pragma experimental "ABIEncoderV2";
 
 contract Arrays {
 
-    uint[3] fixedSimpleArr;
-    uint[] simpleDynamicArr;
+    uint[3] fixedSimpleArr;  // fixed sized normal array
```
Comment (asabya) [inline Arrays/arraySample.sol]: An array is by default 1D array. Please remove `normal`. You can write something like this `array of fixed size` @Soumyanil

File: Arrays/arraySample.sol [hunk 2]
```diff
@@ -3,8 +3,8 @@ pragma experimental "ABIEncoderV2";
...
+    uint[] simpleDynamicArr; // dynamic normal array
```
Comment (asabya) [inline Arrays/arraySample.sol]: `dynamic array` is more effective

File: Arrays/arraySample.sol [hunk 3]
```diff
@@ -3,8 +3,8 @@ pragma experimental "ABIEncoderV2";
 
 contract Arrays {
 
-    uint[3] fixedSimpleArr;
-    uint[] simpleDynamicArr;
+    uint[3] fixedSimpleArr;  // array of fix size
```
Comment (0mkara) [inline Arrays/arraySample.sol]: `// fixed sized array`

File: Arrays/arraySample.sol [hunk 4]
```diff
@@ -3,8 +3,8 @@ pragma experimental "ABIEncoderV2";
...
+    uint[] simpleDynamicArr; // dynamic array
     uint[3][3] fixedSized;   // fixed sized array
```
Comment (0mkara) [inline Arrays/arraySample.sol]: `fixed sized 2D array`

File: Arrays/arraySample.sol [hunk 5]
```diff
@@ -61,12 +61,12 @@ contract Arrays {
         threeDArr = inputArr;
     }
 
-    function getFixedSimpleArr(uint[3] memory simpleArr) public {
-      fixedSimpleArr = simpleArr;
+    function getFixedSimpleArr() public view returns (uint[3] memory){
```
Comment (0mkara) [inline Arrays/arraySample.sol]: Space after parenthesis. `) {`

**Review by 0mkara — CHANGES_REQUESTED** [review]

**Review by 0mkara — APPROVED** [review]

**Pull request merged**